mods/content/karmotrine/machinery/dispenser_presets.dm

```diff
@@ -0,0 +1,29 @@
+/obj/machinery/chemical_dispenser/calicomp
+	name = "\improper CALICOMP 0.9"
+	desc = "A cheap prototype of the CALICOMP bartending assistant. It has a single slot for drinks and six buttons for ingredients. At least it comes with ice!"
+	icon_state = "booze_dispenser"
+	ui_title = "ＣＡＬＩＣＯＭＰ ０.９"
```

Comment:
Is this some unicode stuff or something?

Reply:
that it is! wanted to be fancy so I used fullwidth latin characters, since it's supposed to be made by a vaguely-Japanese tech company in lore i think